CHIPSPVConfig.hh.in

```diff
@@ -42,4 +42,6 @@
 // Non-zero integer if the chipStar is built in debug mode.
 #cmakedefine CHIP_DEBUG_BUILD @CHIP_DEBUG_BUILD@
 
+#cmakedefine CHIP_USE_INTEL_USM
+
```

Comment:
This doesn't require `@CHIP_USE_INTEL_USM` like in the line above?

Reply:
It's not necessary; value is not required for boolean CMake variables. `#cmakedefine X` turns into `#define X` if `X=ON`, and `#undef X` with `X=OFF`.